rest/utilities_testing.go

```diff
@@ -1016,6 +1089,9 @@ type BlipTesterSpec struct {
 
 	// Supported blipProtocols for the client to use in order of preference
 	blipProtocols []string
+
+	// If true, only run the default collection
+	onlyDefaultCollection bool
```

Comment:
Is this actually used anywhere?  I don't see any reads of this property.

Reply:
This is a mistake, I meant to enable this!